tasks/eth/013-fp-upgrade-fjord/VALIDATION.md

```diff
@@ -0,0 +1,167 @@
+      - The address of the Foundation Safe: `0x847B5c174615B1B7fDF770882256e2D3E95b9D92`
+      - The safe hash to approve: `0x530085da61c08016f1625301df4686fcfe40291b98db29a4bf801237ca3098c4`
+    - The using `cast index`, we can verify that:
+      ```shell
+        $ cast index address 0x847B5c174615B1B7fDF770882256e2D3E95b9D92 8
+        0x13908ba1c0e379ab58c6445554ab471f3d4efb06e3c4cf966c4f5e918eca67bd
+      ```
+      and
+      ```shell
+        $ cast index bytes32 0x530085da61c08016f1625301df4686fcfe40291b98db29a4bf801237ca3098c4 0x13908ba1c0e379ab58c6445554ab471f3d4efb06e3c4cf966c4f5e918eca67bd
+        0x6ef669e1a6fdba12525f568b5f98f3455726b23f9e33a5034c9180ca01a3f223 
+      ```
+      And so the output of the second command matches the key above.
+
+
+### `0xc2819DC788505Aac350142A7A707BF9D03E3Bd03` (Council Safe)
+
+State Changes:
+
+#### For the Council:
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000005` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000005` <br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000006` <br/>
+  **Meaning:** The nonce is increased from 5 to 6. The key can be validated by the location of the `nonce` variable in the [Safe's Storage Layout](https://github.com/safe-global/safe-smart-account/blob/v1.3.0/contracts/examples/libraries/GnosisSafeStorage.sol#L17).
+
+
+### `0x847b5c174615b1b7fdf770882256e2d3e95b9d92` (Foundation Safe)
+
+State Changes:
+
+#### For the Foundation:
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000005` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000005` <br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000006` <br/>
+  **Meaning:** The nonce is increased from 5 to 6. The key can be validated by the location of the `nonce` variable in the [Safe's Storage Layout](https://github.com/safe-global/safe-smart-account/blob/v1.3.0/contracts/examples/libraries/GnosisSafeStorage.sol#L17). Note that if this task is simulated before 011 has executed, then the nonce should be increased from 4 to 5.
+
+### `0x24424336F04440b1c28685a38303aC33C9D14a25` (`LivenessGuard`)
+
+State Changes:
+
+#### For the Council:
+
+- **Key:** `0xee4378be6a15d4c71cb07a5a47d8ddc4aba235142e05cb828bb7141206657e27` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x000000000000000000000000000000000000000000000000000000006675b61f` <br/>
+  **Meaning:** This updates the [`lastLive`](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.5.0/packages/contracts-bedrock/src/Safe/LivenessGuard.sol#L36) indicating liveness of an owner that participated in signing. This will be updated to a block timestamp that's close to when this task was executed. Note that the "before" value may be non-zero for signers that have participated in signing.
```

Comment:
```suggestion
  **Meaning:** This updates the [`lastLive`](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.5.0/packages/contracts-bedrock/src/Safe/LivenessGuard.sol#L36) indicating liveness of an owner that participated in signing. This will be updated to a block timestamp that matches the time when this task was executed. Note that the "before" value may be non-zero for signers that have participated in signing.
  
The only other state change is a nonce increment of the account that sent the transaction.
```